### PR TITLE
Initial Travis CI for sdk unittest

### PR DESCRIPTION
**Pull request by Tomcli**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfp-tekton/26)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+matrix:
+  include:
+    - language: python
+      python: "3.6"
```
Comment (ckadner) [inline .travis.yml]: Hi @Tomcli did you try Python 3.7 as well

Comment (Tomcli) [reply .travis.yml]: I can also add that to travis.

**Pull request merged**